.env.example

```
@@ -2,5 +2,22 @@ THE_REGION=ap-northeast-1
+DB_URL=postgresql+asyncpg://<user>:<password>@localhost:5432/<dbname>
+PSQL_TENANT_NAMESPACES=public
+
 # dynamodb
 DYNAMODB_TABLE_ACCOUNTS=dev_x_career_auth_accounts
+
+
+# frontend
+FRONTEND_HOSTNAME=http://localhost:3000
+FRONTEND_URL_PATH_EMAIL_VERIFIED=/auth/email-verified
+FRONTEND_URL_PATH_RESET_PASSWORD=/auth/password-reset
+FRONTEND_TOKEN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The FRONTEND_TOKEN key should go before the FRONTEND_URL_PATH_EMAIL_VERIFIED key

(UnorderedKey)